README.md

```diff
@@ -1,8 +1,30 @@
 # datastore-metadata
-All the LIS Datastore YAML files under one roof.
+All the LIS Datastore YAML files (and some others we consider "metadata") under one roof.
 
-This repository contains the description_[Genus]_[species].yml and README.[collection].yml files in the LIS Datastore.
+This repository contains the description_[Genus]_[species].yml and README.[collection].yml files in the LIS Datastore as well as some other files handled as metadata.
+
+Several types of files should be checked for valid format and contents prior to pushing to the datastore-metadata repository:
+
+```
+  validate.sh description_genus_species YAML_FILE
+  validate.sh description_genus         YAML_FILE
+  validate.sh readme                    YAML_FILE
```

Comment:
This script just kept printing the usage at me, I must be doing something wrong. But, I was able to validate the yaml with executable in the parent directory as per its usage example:
./validate readme.schema.json YAML_FILE

Reply:
thanks, were you running this on the ISU system or on one of the NCGR LIS servers? I think there's a hard-coded path in that script that may be throwing things off if not run on the former, but it should be pretty easily fixed. I put the script in the repository without thinking about it too much, but TBH I didn't actually know it existed until Steven mentioned it...

Reply:
I was on one of our own, I guess that must be why.

Reply:
OK, I think I fixed the hardcoding issue. I think that was indeed causing the behavior you described with it just printing usage. In any case, the alternate (rawer) validation method you figured out should be equivalent, the validate.sh script just does some bit of wrapping of that.